certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7210/CertificateModelFactoryFK7210.java

```diff
@@ -141,10 +138,10 @@ public CertificateModel create() {
   private static ElementSpecification categoryBeraknatNedkomstdatum(
```

Comment:
Should we update the category & question method to Fodelsedatum aswell?

Reply:
Fixed. Also did some search and replace in other classes where the old term was used which I had missed.